src/cli/rustup_mode.rs

```diff
@@ -270,9 +271,9 @@ pub(crate) fn cli() -> Command<'static> {
                 .help("release channel (e.g. +stable) or custom toolchain to set override")
                 .value_parser(|s: &str| {
                     if let Some(stripped) = s.strip_prefix('+') {
-                        ResolvableToolchainName::try_from(stripped).map_err(|e| clap::Error::raw(clap::ErrorKind::InvalidValue, e))
+                        ResolvableToolchainName::try_from(stripped).map_err(|e| clap::Error::raw(clap::error::ErrorKind::InvalidValue, e))
```

Comment:
perhaps import the clap error types locally to this function too, then shorten these to

```suggestion
                        ResolvableToolchainName::try_from(stripped).map_err(|e| raw(InvalidValue, e))
```

Reply:
Done.